src/dotnet/Chat.UI.Blazor/Components/ChatView/Items/MessageContextMenu.razor

```diff
@@ -73,6 +73,10 @@
     }
 
     private async Task InitRequestAuthorId() {
+        if (Session == null)
+            throw new Exception("Session is null");
```

Comment:
Throwing `Exception` is a bad practice: `catch (Exception e) { ... }` catches everything, not just your exception, so ideally you should throw some descendant of `Exception`.